Sources/TIM/Protocols/TIMAuth.swift

```diff
@@ -36,7 +36,7 @@ public protocol TIMAuth {
     ///   - presentingViewController: The view controller which the safari view controller should be presented on.
     ///   - completion: Invoked with access token after successful login (or with error)
     @available(iOS, deprecated: 13)
-    func performOpenIDConnectLogin(presentingViewController: UIViewController, completion: @escaping AccessTokenCallback)
```

Comment:
Could you add an extension to `TIMAuth` with `func performOpenIDConnectLogin(presentingViewController: UIViewController, completion: @escaping AccessTokenCallback)`


Just so we don't break the API

Reply:
I thought that if the param was optional, it would not break the api?

Reply:
Ahh I get what you mean now. It is not possible to define a default value in the protocol. :).

Reply:
I have added an extension function that defines a default parameter for authorizationRequestNonce, but not on without the authorizationRequestNonce param, should we have two separate extension functions?

Reply:
It should actually be without `authorizationRequestNonce: String? = nil`

So

```swift
/// Wrapper of `performOpenIDConnectLogin(presentingViewController:completion:authorizationRequestNonce)` adding nil as default value for authorizationRequestNonce
     func performOpenIDConnectLogin(presentingViewController: UIViewController, completion: @escaping AccessTokenCallback) {
         self.performOpenIDConnectLogin(presentingViewController: presentingViewController, completion: completion, authorizationRequestNonce: nil)
     }
```

Reply:
Ok. Thanks.